web/themes/custom/cloudy/templates/content/location/node--location--search-result.html.twig

```diff
@@ -1,4 +1,36 @@
-<div class="d-flex flex-column align-items-start mb-4">
-  {{ node.field_address_or_entrance.address_line1 }}{% if node.field_address_or_entrance.address_line2 %}, {{ node.field_address_or_entrance.address_line2 }}{% endif %}{% if node.field_address_or_entrance.locality %}, {{ node.field_address_or_entrance.locality }}{% endif %}{% if node.field_address_or_entrance.administrative_area %}, {{ node.field_address_or_entrance.administrative_area }}{% endif %}{% if node.field_address_or_entrance.postal_code %}, {{ node.field_address_or_entrance.postal_code }} {% endif %}
-  {{ content }}
+{%
+  set classes = [
```

Comment:
These aren't indexed yet, but I cleaned up the search result to match the other location templates.